src/frontend-build.sh

```
@@ -256,18 +256,18 @@ export GIT_COMMIT=$(git rev-parse HEAD)
 if [ $IS_PR = true ]; then
   verifyDependencies
 
-  docker  build -t "${IMAGE}:${IMAGE_TAG}" $APP_ROOT -f $APP_ROOT/Dockerfile
+  docker build --pull -t "${IMAGE}:${IMAGE_TAG}" $APP_ROOT -f $APP_ROOT/Dockerfile
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)